app/src/Model/Entity/User.php

```diff
@@ -43,4 +44,19 @@ class User extends Entity
     protected $_hidden = [
         'password',
     ];
+
+    /**
+     * Hash password.
+     *
+     * @param string $password User password.
+     * @return string|null
+     */
+    protected function _setPassword(string $password): ?string
+    {
+        if (strlen($password) > 0) {
+            return (new DefaultPasswordHasher())->hash($password);
```

Comment:
Since we hash the password, it's better to fix the UsersSeed contain the plaintext password https://github.com/combimauri/tutorial-cakephp/blob/632197554fc090909fef6e9bef9db270a458b378/app/config/Seeds/UsersSeed.php#L27

Please check.